# Stats from a single tag reading 
#   when was read
#   signal strength (dBm)
#   wave phase  (degrees)
#   wave frequency (KHz)


time   timestamp
int32  rssi
int32  phase
int32  frequency
